mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -463,6 +471,53 @@ is set to true.`,
 							},
 							Description: `Configuration of Query Insights.`,
 						},
+						"password_validation_policy": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"min_length": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										ValidateFunc: validation.IntBetween(0, 2147483647),
+										AtLeastOneOf: passwordValidationPolicyKeys,
```

Comment:
Question: Is `AtLeastOneOf` still valid for this field + all other fields under `password_validation_policy`?

My concerns:
1. Is it necessary to set at least one of the fields in `enable_password_policy` to be set to configure the `password_validation_policy`?
My guess that we originally need this `AtLeastOneOf` is that we try to avoid allowing users to set an empty block but since we already have `enable_password_policy` as required, we may not need this.
3. `enable_password_policy` can be set as `False` and all other fields under `password_validation_policy` can't be set in this case. However, with `AtLeastOneOf`, users still need to set at least one field.

Reply:
You are right, we usually don't use `AtLeastOneOf` when a required field is already included. Fixed it